ADAL/src/request/ADAuthenticationRequest+WebRequest.m

```diff
@@ -332,16 +332,13 @@ - (void)requestWithServer:(NSString *)authorizationServer
     SAFE_ARC_RELEASE(webRequest);
 }
 
-//Used for the callback of obtaining the OAuth2 code:
-static volatile int sDialogInProgress = 0;
-
 //Ensures that a single UI login dialog can be requested at a time.
 //Returns true if successfully acquired the lock. If not, calls the callback with
 //the error and returns false.
 - (BOOL)takeExclusionLockWithCallback: (ADAuthorizationCodeCallback) completionBlock
```

Comment:
I found that this function cannot be directly used for the broker case, because it accepts a different completionBlock type and do additional things in the function :P